### ISSUE TITLE
[FEATURE]

### ISSUE THREAD

**Issue by Aadi-mone** [booksaw/BetterTeams#263]
Please add a feature in the config where we can enable the PVP, in some "worlds"
when we specify some worlds, then the PVP is enabled there, even if it's disabled by the team leaders or world guard.

example:
```yml

pvp-enabled-worlds:
- pvp
- pvp2
```

**Comment by bobby29831** [comment]
Having plugins override the behaviors of other plugins is not good practice. It can become immensely confusing. This is not the intended use for this plugin, and would make it immensely difficult to narrow down as the cause for issues if they arise.

I would not recommend this feature be implemented for that reason. You can accomplish toggling these settings through other plugins like World Guard.

**Comment by Aadi-mone** [comment]
Yes, you're right, I tried world guard with it, but for some reason, it doesn't work, which is why I am requesting this.
and how does a plugin override another plugin? better teams it self making a new feature where we can enable the entire pvp for all teams present in better teams! 
and thank you for your response 

Greetings,
~ Adarsh A

**Comment by booksaw** [comment]
Instead of making a new feature, I think it would be a better idea to fix the already existing feature.
What is wrong with the worldgaurd flag?

**Comment by Aadi-mone** [comment]
yes, as in better teams there is an option to disable PVP, but we can't enable it, and how come the world guard works when the player does /team PVP and one player is op-ed and can't be changed, so it will override world guard flags tho. 

and thank you for your response

Greetings,
~ Adarsh A

**Comment by booksaw** [comment]
Im not sure what you mean?

**Comment by Aadi-mone** [comment]
Let me explain:
The option:
```yml
disablecombat: true
```
was removed on a feature update, and because of that, the server admins/owners can't completely enable PVP In the config, which is a problem, so I want the option:
```yml
disablecombat: true
``` 
to come back, in which we can enable the PVP even if the team owners do `/team pvp`
Hoping that you understood :)

Greetings,
~ Adarsh

**Comment by booksaw** [comment]
disableCombat has not been removed?

**Comment by Aadi-mone** [comment]
well, if it's not removed then it's a bug xD
and please check #235 
I enabled it still PVP not being disabled, anyways a new version is required!

**Comment by Aadi-mone** [comment]
well, if it's not removed then it's a bug xD
and please check #235
I enabled it still PVP not being disabled, anyways a new version is required!

Kind Regards,
~ Adarsh

**Comment by Aadi-mone** [comment]
well, if it's not removed then it's a bug xD
and please check #235
I have enabled it still PVP is not being enabled, anyways a new version is required!

Kind Regards,
~ Adarsh

**Comment by booksaw** [comment]
This issue is being closed for inactivity. If you are still having problems feel free to open a new issue linking this one, make sure you answer any questions asked in the new issue.
